build-system/test-configs/karma.conf.js

```diff
@@ -80,6 +80,10 @@ module.exports = {
     },
   },
 
+  junitReporter: {
+    useBrowserName: false,
+  },
```

Comment:
Can this be deleted now that you're doing this in `runtime-test-base.js`?

Reply:
Definitely